modules/pages.xqm

```diff
@@ -49,32 +49,48 @@ function pages:load($node as node(), $model as map(*), $publication-id as xs:str
             pages:last-modified($publication-id, $document-id, $section-id)
         else 
             ()
-    let $content := map {
-        "data":
-            if (exists($publication-id) and exists($document-id)) then
-                pages:load-xml($publication-id, $document-id, $section-id, $view, $ignore)
-            else (),
-        "publication-id": $publication-id,
-        "document-id": $document-id,
-        "section-id": $section-id,
-        "view": $view,
-        "base-path":
-            (: allow for pages that do not have $config:PUBLICATIONS?select-document defined :)
-            (: ... TODO: I do not see any such cases in config:PUBLICATIONS! Check if OK to remove this entry? - JW :)
-            if (exists($publication-id) and map:contains(map:get($config:PUBLICATIONS, $publication-id), 'base-path')) then
-                map:get($config:PUBLICATIONS, $publication-id)?base-path($document-id, $section-id)
-            else (),
-        "odd": if (exists($publication-id)) then map:get($config:PUBLICATIONS, $publication-id)?transform else $config:odd-transform-default
-    }
-
+    let $if-modified-since := try { request:get-attribute("if-modified-since") => parse-ietf-date() } catch * { () }
+    let $should-return-304 := 
+        if (exists($last-modified) and exists($if-modified-since)) then
+            $if-modified-since gt $last-modified
```

Comment:
As per [rfc2616](https://datatracker.ietf.org/doc/html/rfc2616#page-130) clients SHOULD set If-Modified-Since to a previously received Last-Modified. So using `ge` (greater or equal) instead of `gr` (greater) here seems desirable. Otherwise most requests will still get a 200 instead of a 304.

Reply:
@agh2342 Thank you! I've now switched to `ge`.